pyproject.toml

```diff
@@ -301,27 +300,6 @@ dbt = [
 adbc = [
     "adbc-driver-postgresql>=1.6.0"
 ]
-# NOTE: depedencies required to lint and test docs and examples
-docs = [
-    "SQLAlchemy>=1.4.0",
-    "pymysql>=1.1.0,<2",
-    "pypdf2>=3.0.1,<4",
-    "pydoc-markdown>=4.8.2,<5",
-    "dbt-core>=1.5.0",
-    "dbt-duckdb>=1.5.0",
-    "pymongo>=4.3.3",
-    "pandas>=2.1.4",
-    "alive-progress>=3.0.1",
-    "pyarrow>=16.0.0",
-    "psycopg2-binary>=2.9",
-    "lancedb>=0.8.2 ; python_version < '3.13'",
-    "openai>=1.45",
-    "connectorx>=0.4.2 ; python_version >= '3.10'",
-    "modal>=0.64.170",
-    # google secrets provider
-    "google-api-python-client>=1.7.11",
-    "google-auth-oauthlib>=1.0.0,<2",
-]
 
 [project.entry-points.dlt]
```

Comment:
pls clean up pytest.ini entry

Reply:
I removed the docs paths and test patterns, I think that should be sufficient, right?